src/libs/actions/Task.js

```diff
@@ -398,7 +417,7 @@ function editTaskAndNavigate(report, ownerEmail, ownerAccountID, {title, descrip
             editedTaskReportActionID: editTaskReportAction.reportActionID,
             assigneeChatReportActionID: optimisticAssigneeAddComment ? optimisticAssigneeAddComment.reportAction.reportActionID : 0,
         },
-        {optimisticData, successData, failureData},
+        {optimisticData, failureData},
```

Comment:
Are we sure we don't need to pass successData at all here? I thought I remember something like if successData didn't need to be different than optimisticData we should still pass `successData: optimisticData`?

Reply:
There are other calls to API.write that omit the `successData` and after testing everything worked fine so I don't think we need to pass that.

Since I'm not 100% sure let's ask Jack since he introduced these specific lines.

@thienlnam, do you remember if there is a reason for empty array `successData` and if it's ok to remove it?

Reply:
Ah yeah the empty successData is okay to remove

> I thought I remember something like if successData didn't need to be different than optimisticData we should still pass successData: optimisticData?

We don't do this because the data is set optimistically on the device, and then on success it sends pusher updates to all clients to update the onyxData. So in this case, if we had successData, it would merge the same data as optimisticData on success, but then we are also getting the pusher data which is the same as the optimisticData so nothing really changes

Reply:
coolio thanks for the clarification